fuel-core/src/schema.rs

```diff
@@ -42,6 +42,19 @@ pub struct Subscription(tx::TxStatusSubscription);
 
 pub type CoreSchema = Schema<Query, Mutation, Subscription>;
 
+fn check_pagination_parameters<T>(
```

Comment:
Can the name be more descriptive + potentially a comment above the function?
I know that it `checks` parameters but for what?, their validity, their combo-validity, etc?

Reply:
Yes I can change that to be more descriptive